enterprise_access/apps/content_assignments/utils.py

```diff
@@ -0,0 +1,55 @@
+"""
+Utils for content_assignments
+"""
+from datetime import datetime, timedelta
+
+from dateutil import parser
+from pytz import UTC
+
+from enterprise_access.apps.content_assignments.constants import (
+    BRAZE_ACTION_REQUIRED_BY_TIMESTAMP_FORMAT,
+    START_DATE_DEFAULT_TO_TODAY_THRESHOLD_DAYS
+)
+
+
+def is_within_minimum_start_date_threshold(today, start_date):
```

Comment:
nit: could we rename today to curr_date or now or something else? 

Why? Feels ambiguous? it could possibly not really be today anymore at execution time?